test/index.test.ts

```diff
@@ -286,7 +286,34 @@ describe('HipThrusTS', () => {
   });
 */
   describe('Hipthrusts functional only', () => {
```

Comment:
forgot to mention...
- [ ] add test coverage for multi-stage operators

Reply:
- [ ] also need a todo for adding test coverage for weird operators that can return bool or void (e.g. authorizers and doWork)
- [ ] also todo for testing stages that do NOT snowball context (e.g. sanitizers and respond)